libraries/node-core-library/src/IPackageJson.ts

```diff
@@ -104,6 +104,12 @@ export interface IPackageJson {
    */
   tsdoc?: IPackageJsonTsdocConfiguration;
 
+  /**
+   * The path to the tsdoc file metadata.
```

Comment:
>The path to the tsdoc file metadata. [](start = 5, length = 36)

"The path to the TSDoc metadata file".  Also since this is an emerging standard, it might be useful to include the URL for the GitHub issue that describes this feature